recipes/caiman/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "CaImAn" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flatironinstitute/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ab272a4cd8ed4200fd844c7a9fa5770257313a8f6eb658ce57a82577f0ca3e75
+
+build:
+  number: 0
+  skip: True  # [not py==36]
```

Does this need to limit to only 3.6? Normally 3.7 is okay as well:
```suggestion
  skip: True  # [py<36]
```